Userland/Libraries/LibWeb/HTML/TraversableNavigable.h

```diff
@@ -82,6 +82,9 @@ class TraversableNavigable final : public Navigable {
     Page& page() { return m_page; }
     Page const& page() const { return m_page; }
 
+    String window_handle() const { return m_window_handle; }
+    void set_window_handle(String window_handle) { m_window_handle = move(window_handle); }
```

Comment:
as I see `TraversableNavigable::window_handle()` is not used anywhere but it is going to be a replacement for browsing context's window handle in WebDriver, right?

Reply:
That's correct yeah. We'll need to replace the search for a browsing context by handle to a traversable navigable. This PR already breaks the naive open window support in Web driver, we'll need a follow up to fix it